scripts/config/lib/nginx_config_util.rb

```diff
@@ -44,4 +44,16 @@ def self.match_redirects(redirects, uri)
 
     false
   end
+
+  def self.interpolate(string, vars)
+    regex = /\${(\w*?)}/
+
+    string.scan(regex).inject(string) do |acc, capture|
+      var_name = capture.first
+      value = vars[var_name] if vars
+      acc.sub!("${#{var_name}}", value) if value
```

Comment:
Would you want to escape just the `static.json` config key or also expect the env var to escape too? So escape on line 52 or 54?